README.md

```diff
@@ -46,7 +46,7 @@ That's it!
 
 While logged in as an admin, click the "Redirects" button. A list of redirects appears, initially empty. Add as many redirects as you like. The "from" URL must begin with a `/`. The "to" URL may be anything and need not be on your site. The "description" field is for your own convenience.
 
-By default a redirect includes any query string on incoming requests when matching for redirection.  You can toggle the 'Ignore query string when matching.' option in a redirect definition to ignore query strings on incoming requests and only match on the base URL path.
+By default a redirect includes any query string (the `?` and whatever follows it, up to but not including any `#`) on incoming requests when matching for redirection. You can toggle the "ignore query string when matching." option in a redirect definition to ignore query strings on incoming requests and only match on the base URL path. A redirect that does not use this option will always match first, so you can match various specific query strings and then have a fallback rule for other cases.
```

Comment:
> "ignore query string when matching."

We should either leave as capitalized to match the UI or remove the period to suggest the sentence as a quotation.